test/pytest/test_bridge.py

```diff
@@ -477,6 +477,49 @@ async def test_fslist1_notexist(transport):
         reply_keys={'message': "[Errno 2] No such file or directory: '/nonexisting'"})
 
 
+@pytest.mark.asyncio
+async def test_fsreplace1(transport):
+    tempdir = tempfile.TemporaryDirectory()
```

Comment:
You'd want to use this as a context manager, except that you should actually use the `tmp_path` fixture for this... 
https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

Reply:
Done, thanks!